components/badge/stories/badge.stories.js

```diff
@@ -75,18 +80,32 @@ export default {
 	},
 };
 
-export const Default = Template.bind({});
-Default.args = {
-	label: "Badge",
-};
-
-export const WithIcon = Template.bind({});
-WithIcon.args = {
-	iconName: "Info",
-	label: "Badge",
+const BadgeGroup = ({
```

Comment:
For sync discussion: what are our thoughts on the name BadgeGroup?, I also do something similar in checkbox. I was concerned it might get confused with components like TagGroup but struggled to come up with a better name.